sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -137,7 +136,7 @@ Fixed the issue that the second call of `ServiceBusReceiverClient.complete` is s
 - Upgraded `azure-core` dependency to `1.15.0`.
 - Upgraded `azure-core-amqp` dependency to `2.0.4`.
 
-## 7.2.0-beta.1 (2021-03-18)
```

Comment:
Sorry, what I meant was the `-beta.1` meant to be removed since the release on that day was `7.2.0-beta.1`: https://mvnrepository.com/artifact/com.azure/azure-messaging-servicebus/7.2.0-beta.1

(Actually, a lot of these old CHANGELOG release dates look wrong)

Reply:
We still need to keep track of all releases in the changelog. If we remove this, we'll lose track of the 7.2.0-beta.1 release changes.

Reply:
Done